dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -752,8 +753,19 @@ else if (field.getFieldType().equals(ESMappingConstants.FIELD_ELASTIC_TYPE_DATE)
 						contentletMap.put(keyName, valueObj);
 						contentletMap.put(keyNameText, numFormatter.format(valueObj));
 					} else {
-						contentletMap.put(keyName, valueObj);
-						contentletMap.put(keyNameText, valueObj.toString());
+					    if (UtilMethods.isSet(valueObj) && valueObj instanceof Date){
+                            try {
+                                String datetimeString = datetimeFormat.format(valueObj);
+                                contentletMap.put(keyName, elasticSearchDateTimeFormat.format(valueObj));
+                                contentletMap.put(keyNameText, datetimeString);
+                            } catch(Exception ex) {
```

Comment:
It is worth logging this ex? @nollymar

Reply:
It isn't. In case it doesn't work , the old behavior should be kept